repos/system_upgrade/el7toel8/actors/checkselinux/tests/test_checkselinux.py

```diff
@@ -2,31 +2,63 @@
 from leapp.models import (SystemFacts, CheckResult, SELinux, SelinuxPermissiveDecision,
                           SelinuxRelabelDecision)
 
+# import needed just to be able to create valid model, but not important for test
+from leapp.models import FirewallStatus, Firewalls
+
+
+# FIXME: fix the file properly regarding the fix of the issue:
```

Comment:
the line below.

Reply:
> Running the tests on my machine does not end well:
> https://gist.github.com/bocekm/fc30c4b680467ed2cc98b0a9e0f8d560

Yes, it's because you ran the tests on your host. You need to run test on a machine as root because some tests touch the system.